src/SourceBuild/content/repo-projects/installer.proj

```diff
@@ -81,4 +84,63 @@
     <EnvironmentVariables Include="CLIBUILD_SKIP_BUNDLEDDOTNETTOOLS=true" />
   </ItemGroup>
 
+  <Target Name="GetInputsOutputForCreatePrivateSourceBuiltArtifactsArchive"
+          DependsOnTargets="DetermineSourceBuiltSdkVersion">
+    <!-- Inputs: Packages to include in the tarball -->
+    <ItemGroup>
+      <ArtifactsPackageToBundle Include="$(ArtifactsShippingPackagesDir)**;
+                                         $(ArtifactsNonShippingPackagesDir)**"
+                                Condition="!$([System.String]::Copy('%(Identity)').EndsWith('.symbols.nupkg'))" />
+      <ReferencePackageToBundle Include="$(ReferencePackagesDir)**"
+                                Condition="!$([System.String]::Copy('%(Identity)').EndsWith('.symbols.nupkg'))" />
```

Comment:
This shouldn't be necessary right?

Reply:
My comment was more about we should never be producing symbols for the ReferencePackages in the first place.  There should never be symbols packages in the ReferencePackagesDir